cmd/version.go

```diff
@@ -16,6 +17,9 @@ func getCmdVersion(globalState *globalState) *cobra.Command {
 		Long:  `Show the application version and exit.`,
 		Run: func(_ *cobra.Command, _ []string) {
 			printToStdout(globalState, fmt.Sprintf("k6 v%s\n", consts.FullVersion()))
+			for path, version := range modules.GetJSModuleVersions() {
+				printToStdout(globalState, fmt.Sprintf("extension %s %s\n", path, version))
+			}
```

Comment:
k6 supports both JS and output extensions. So we should make sure that both are shown in this output (same goes for the `k6 run` output).

You can test by building a binary with an output extension, e.g.:
```
XK6_K6_REPO=github.com/HarrisChu/k6 xk6 build master \
  --with github.com/vesoft-inc/k6-plugin \
  --with github.com/grafana/xk6-output-influxdb
```

It should be a matter of doing something similar as in `js/modules/modules.go` and calling `getPackageVersion(mod)` from [`output.RegisterExtension()`](https://github.com/grafana/k6/blob/e14f14d416b0af4e495e6f565a2a9787a73ade54/output/extensions.go#L27). You'll probably want to move `getPackageVersion()` somewhere where it's accessible by both (maybe `lib/extensions.go`?).